dns_stealth_tool.py

```
@@ -0,0 +1,715 @@
+    def check_dns_health(self, domain):
+        """فحص صحة DNS"""
+        try:
+            health_checks = []
+            
+            # فحص A record
+            try:
+                resolver = dns.resolver.Resolver()
+                resolver.nameservers = ['8.8.8.8']
+                a_records = resolver.resolve(domain, 'A')
+                a_status = 'healthy' if len(a_records) > 0 else 'no_a_record'
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)